tastytrade/dxfeed/timeandsale.py

```diff
@@ -0,0 +1,52 @@
+from dataclasses import dataclass
+
+from .event import Event
+
+
+@dataclass
+class TimeAndSale(Event):
+    """
+    A Quote event is a snapshot of the best bid and ask prices, and other fields that change with each quote.
```

Comment:
Can you update the comment for the event type?

Reply:
Good catch. I updated the comment with the description frojm DxFeed's knowledge base.